test/unit/signup/account_manager_test.rb

```diff
@@ -126,23 +126,29 @@ class ProviderAccountManagerTest < Signup::AccountManagerTest
       assert_equal 'API', account.first_service!.name
     end
 
-    test 'creating a provider will create default Backend if api_as_product is disabled' do
+    test 'creating a provider will create default Backend regardless of api_as_product' do
```

Comment:
This is not "regardless APIAP". This is actually "without APIAP".

Reply:
Right sorry, bcz I tried refactoring in one and then I came back to the previous version and forgot the change this :+1: